apps/meteor/app/apps/server/converters/messages.js

```diff
@@ -243,6 +244,14 @@ export class AppMessagesConverter {
 		);
 	}
 
+	async convertSystemMessage(message) {
```

Comment:
I would say that we shouldn't need a new method for system messages, considering that the values don't seem to vary that much. What was your thought on adding a new method?

Reply:
An interesting "side-effect" of using the same converter method is that we can use the same  `messageEvent` method from the listener class